cypress/integration/dishes.js

```diff
@@ -31,6 +31,23 @@ describe('Lunch app. Dishes order.', function() {
             dishesPage.getFirstDish().click();
             dishesPage.getOrderButton().should('not.have.attr', 'disabled', 'disabled');
             dishesPage.getOrderedTooltipsIcons().first().contains('clear');
-        }
+        }        
+    })
+
+    it('Create a new user', function() {
+        const date = new Date();
+        const newUserName = 'Vardas';
+
+        if (date.getDay() < 5) {
+            const dayName = common.getDay(date.getDay() + 1);
+            dishesPage.visitDay(dayName);
+            dishesPage.getAddButton().click();
+            dishesPage.getCreateNewUserButton().click();
+            dishesPage.getNameFieldInCreateUserForm().type(newUserName);
+            dishesPage.getEmailFieldInCreateUserForm().type('pastas@gmail.com');
```

Comment:
If newUserName constant is created for name, why not create newUserEmail constant for email?